tools/clang/test/CodeGenSPIRV/ternary-op.short-circuited-cond-op.sampler-state.hlsl

```diff
@@ -0,0 +1,10 @@
+// RUN: %dxc -T ps_6_0 -E main -HV 2021 -fcgl  %s -spirv | FileCheck %s
```

Comment:
This will fail since dxc will report an error and return a non-zero value. You'll need to replace it with
```suggestion
// RUN: not %dxc -T ps_6_0 -E main -HV 2021 -fcgl  %s -spirv 2>&1 | FileCheck %s
```